packages/pn-commons/src/components/NotificationDetail/NotificationDetailTimelineStep.tsx

```diff
@@ -187,6 +198,7 @@ const NotificationDetailTimelineStep = ({
     undefined,
     <Box data-testid="moreLessButton">
       <ButtonNaked
+        id="show-quantity"
```

Comment:
@leleOFA cambia in "more-less-timeline-step"

Reply:
fatto